src/darwin/Framework/CHIPTests/MTROTAProviderTests.m

```diff
@@ -46,7 +46,9 @@
 static const uint16_t kTimeoutWithUpdateInSeconds = 60;
 static const uint64_t kDeviceId1 = 0x12341234;
 static const uint64_t kDeviceId2 = 0x12341235;
+#ifdef ENABLE_REAL_OTA_UPDATE_TESTS
```

Comment:
we can add the ifdef ENABLE_REAL_OTA_UPDATE_TESTS around kOnboardingPayload3 as well.

Reply:
Done.